jenkins/spark-premerge-build.sh

```diff
@@ -38,6 +38,9 @@ tar zxf $SPARK_HOME.tgz -C $ARTF_ROOT && \
     rm -f $SPARK_HOME.tgz
 
 mvn -U -B $MVN_URM_MIRROR clean verify -Dpytest.TEST_TAGS=''
+mvn -U -B $MVN_URM_MIRROR -Pspark301tests test -Dpytest.TEST_TAGS=''
```

Comment:
Unless we set SPARK_HOME to point to a 3.0.1 cluster running the python tests again is not going to help. We should probably disable them until we can make it work. Youc an disable just the python tests by not setting SPARK_HOME

```
env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark301tests test
```